packages/villager-info/src/VillagerInfo.js

```diff
@@ -1,25 +1,115 @@
-import { LitElement, html } from 'lit-element';
+import { LitElement, html, nothing } from 'lit-element';
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { localize, LocalizeMixin } from '@lion/localize';
+import { LionButton } from '@lion/button';
 
 import { styles } from './VillagerInfo.styles.js';
 
-export class VillagerInfo extends LitElement {
+const LOCALE_KEY = 'villager-info';
+
+export class VillagerInfo extends LocalizeMixin(
+  ScopedElementsMixin(LitElement)
+) {
+  static get scopedElements() {
+    return {
+      'lion-button': LionButton,
+    };
+  }
+
   static get styles() {
     return styles;
   }
 
+  static get localizeNamespaces() {
+    return [
+      {
+        [LOCALE_KEY]: (locale) => {
+          const namespaces = {
+            'en-GB': () => import('../translations/en-GB.js'),
+            'es-ES': () => import('../translations/es-ES.js'),
+          };
+
+          return (namespaces[locale] || namespaces['en-GB'])();
+        },
+      },
+      ...super.localizeNamespaces,
+    ];
+  }
+
+  static get events() {
+    return {
+      add_my_villager: 'add-my-villager',
+      add_wishlist: 'add-wishlist',
+    };
+  }
+
   static get properties() {
     return {
       villager: { type: Object },
+      showWishlistButton: { type: Boolean },
+      isInWishlist: { type: Boolean },
+      showMyVillagersButton: { type: Boolean },
+      isInMyVillagersList: { type: Boolean },
     };
   }
 
   constructor() {
     super();
 
     this.villager = {};
+    this.showWishlistButton = true;
+    this.isInWishlist = false;
+    this.showMyVillagersButton = true;
+    this.isInMyVillagersList = false;
+  }
+
+  isVillagerEmpty(villager) {
+    return Object.keys(villager).length === 0;
+  }
+
+  onClickMyVillagersButton() {
+    this.dispatchEvent(new CustomEvent('add-my-villager'));
```

Comment:
Please update this an use the static method

Reply:
Done